app/actions/GovernanceActions.js

```diff
@@ -686,7 +686,15 @@ export const updateVoteChoice = (
     });
 
     // cast vote into pi server
-    await pi.castVotes({ piURL, votes });
+    const response = await pi.castVotes({ piURL, votes });
+    const voteCastError = response.data.receipts.find(
```

Comment:
```suggestion
    const { error } = response.data.receipts.find(
```

Reply:
If there are no receipts errors, this will try to access the prop `error` of undefined, see?

Reply:
you could add `|| {}` at the end of this _find_ function.

Reply:
ie `find(.....) || {};`

Reply:
I think this is a bit confusing in terms of readability